extide/gradle/netbeans-gradle-tooling/src/main/java/org/netbeans/modules/gradle/tooling/NbProjectInfoBuilder.java

```diff
@@ -172,7 +180,8 @@ private void detectProjectMetadata(NbProjectInfoModel model) {
         model.getInfo().put("project_buildDir", project.getBuildDir());
         model.getInfo().put("project_projectDir", project.getProjectDir());
         model.getInfo().put("project_rootDir", project.getRootDir());
-        model.getInfo().put("gradle_user_home", project.getGradle().getGradleHomeDir());
+        model.getInfo().put("gradle_user_home", project.getGradle().getGradleUserHomeDir());
```

Comment:
Passing `gradleHomeDir` instead of userdir caused the `GradleModuleFileCache21` to work over **gradle distribution** directory rather than **user cache** directory, effectively disabling the accessor.

Reply:
Nice catch!